signing/signer.go

```diff
@@ -0,0 +1,58 @@
+package signing
+
+import (
+	"errors"
+	"github.com/spacemeshos/ed25519"
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+type BlsFetcher interface {
+	FetchBlsKey(edKey []byte) []byte
+}
+
+type ActivationValidator interface {
+	IsActive(blsKey []byte) bool
+}
+
+type Signer interface {
```

Comment:
Why do you need to define the interface here, instead of where you use the interface like the go convention?